accounts/serializers.py

```diff
@@ -51,19 +51,16 @@ class Meta:
             'state',
             'density',
             'extra',
+            'date_created',
```

Comment:
Not sure what you need for your use case, but when I recently added `date_created` to the Response model, on the Response serializer, I changed the field name to 'created_on' to match what your randomizer was expecting.  
 `created_on = serializers.DateTimeField(read_only=True, source='date_created')`

Reply:
I saw that - thanks! I figured it might be simpler, all else equal, just to use the same date_created name but not switch it in the serializer, but am open to doing it either way. Nothing else on Lookit (in the ember-lookit-frameplayer/lookit-api) makes use of the date created info for users or demographic data, so we don't have anything we need to match, and it's all the same to me as I get our analysis code that uses the API working.